pyproject.toml

```diff
@@ -4,7 +4,7 @@ requires = [
     "ninja",
     "packaging",
     "setuptools >= 49.4.0",
-    "torch == 2.1.2",
+    "torch == 2.2.0",
```

Comment:
I think this one is problematic since the file is shared between NVIDIA and AMD GPUs... 😓

Reply:
the key is that `pip install vllm` won't try to force torch==2.1.2 and break the environment

Reply:
After second look, I realized that actually `pyproject.toml` is not used for the ROCm backend.